agent/pipe/port.go

```diff
@@ -935,6 +937,10 @@ func readAllContainerPorts(pid int, existPairs map[string]*InterceptPair) ([]*In
 			}
 		}
 
+		if link.Type() == "vxlan" {
+			hasVxlan = true
+			log.Debug("vxlan")
```

Comment:
Need more clear debug message than just `vxlan`. Maybe `VXLAN detected in container` or something more verbose.